src/main/java/com/exasol/validation/PlatformValidatorFactory.java

```diff
@@ -0,0 +1,30 @@
+package com.exasol.validation;
+
+import com.exasol.GitRepository;
+import com.exasol.ReleasePlatform;
+
+/**
+ * Responsible for instantiation of {@link PlatformValidator}s.
+ */
+public final class PlatformValidatorFactory {
+    private PlatformValidatorFactory() {
+        // prevent instantiation
+    }
+
+    /**
+     * Create a new instance of the {@link PlatformValidator} depending on the {@link ReleasePlatform}.
+     *
+     * @param repository {@link GitRepository} with the project to release
+     * @param platform release platform
+     * @return new instance of {@link PlatformValidator}
+     */
+    public static PlatformValidator createProjectValidator(final GitRepository repository,
+            final ReleasePlatform platform) {
+        if (platform == ReleasePlatform.GITHUB) {
```

Comment:
Please write a ticket to remove coupling here later. Use the Java Service Loader to get rid of hard-coded switches. Check the service loader implementation for the VS dialects as an example.